runtime/src/lib.rs

```diff
@@ -116,6 +116,8 @@ pub const DAYS: BlockNumber = HOURS * 24;
 pub const MILLICENTS: Balance = 1_000_000_000;
 pub const CENTS: Balance = 1_000 * MILLICENTS;
 pub const DOLLARS: Balance = 100 * CENTS;
+// LIT target price 100 dollar
+pub const LITS: Balance = 100 * DOLLARS;
```

Comment:
what is target price?

Reply:
need figure out decimal definition and how does it map to fee.